### PR TITLE
CLDR-15924 finalize changes to person names

### PR DESCRIPTION
**Pull request by macchiati**
CLDR-15924

- [ ] This PR completes the ticket.

### CODE REVIEW

**Comment by macchiati** [comment]
If the tests pass, I think we can go ahead and merge this much, then follow up with a different PR for Mike's changes.

File: common/main/ff.xml [hunk 1]
```diff
@@ -821,4 +821,8 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 			<nostr>Alaa:A</nostr>
 		</messages>
 	</posix>
+	<personNames>
+		<nameOrderLocales order="givenFirst">und ff</nameOrderLocales>
+		<nameOrderLocales order="surnameFirst">ko vi yue zh</nameOrderLocales>
+	</personNames>
```
Comment (srl295) [inline common/main/ff.xml:827]: Just curious… what value does this add to add this to several locales?  

There doesn't seem to be any documentation for these elements in the spec.  But does does `<nameOrderLocales order="givenFirst">und</nameOrderLocales>` already apply to all locales?  Why add this to, say, ff?

The "surnameFirst" data is also in root, is there a reason to repeat it here?

File: tools/cldr-code/src/main/resources/org/unicode/cldr/tool/modify_config.txt [hunk 2]
```diff
@@ -1,25 +1,151 @@
...
+locale=te ; action=add ; new_path=//ldml/personNames/nameOrderLocales[@order="surnameFirst"]; new_value=ko te vi yue zh
+locale=ti ; action=add ; new_path=//ldml/personNames/nameOrderLocales[@order="surnameFirst"]; new_value=ko te vi yue zh
+locale=am ; action=add ; new_path=//ldml/personNames/nameOrderLocales[@order="surnameFirst"]; new_value=ko vi yue zh
+locale=ar ; action=add ; new_path=//ldml/personNames/nameOrderLocales[@order="surnameFirst"]; new_value=ko vi yue zh
+locale=ast ; action=add ; new_path=//ldml/personNames/nameOrderLocales[@order="surnameFirst"]; new_value=ko vi yue zh
+locale=az ; action=add ; new_path=//ldml/personNames/nameOrderLocales[@order="surnameFirst"]; new_value=ko vi yue zh
+locale=be ; action=add ; new_path=//ldml/personNames/nameOrderLocales[@order="surnameFirst"]; new_value=ko vi yue zh
+locale=bg ; action=add ; new_path=//ldml/personNames/nameOrderLocales[@order="surnameFirst"]; new_value=ko vi yue zh
+locale=bn ; action=add ; new_path=//ldml/personNames/nameOrderLocales[@order="surnameFirst"]; new_value=ko vi yue zh
+locale=br ; action=add ; new_path=//ldml/personNames/nameOrderLocales[@order="surnameFirst"]; new_value=ko vi yue zh
```
Comment (srl295) [inline tools/cldr-code/src/main/resources/org/unicode/cldr/tool/modify_config.txt:53]: same comment. maybe a better data model is needed if this information is repeated in each locale?

**Review by srl295 — COMMENTED** [review]
Changes seem OK, I do have questions about the repetitiveness of the data model.  

Elements seem to be undocumented.

**Review by srl295 — APPROVED** [review]

**Review by srl295 — APPROVED** [review]

File: tools/cldr-code/src/main/resources/org/unicode/cldr/tool/modify_config.txt (hunk 2 above)
Comment (macchiati) [reply tools/cldr-code/src/main/resources/org/unicode/cldr/tool/modify_config.txt:53]: There is a lot of commonality among the locales, but each one is a separate decision, based on what is customary in the locale.

File: common/main/ff.xml (hunk 1 above)
Comment (macchiati) [reply common/main/ff.xml:827]: These are in the design doc (haven't made it into LDML yet, but will). They say:

For formatting these languages in this locale, use the surnameFirst or givenFirst patterns, respectively.

**Comment by macchiati** [comment]
Just fixed a last (hopefully) typo.

**Review by srl295 — APPROVED** [review]

File: common/main/ff.xml (hunk 1 above)
Comment (srl295) [reply common/main/ff.xml:827]: Perhaps the spec could link to the design doc as an interim step? 

My specific question was if `und ff` was any different than `und` where ff is not mentioned.

**Pull request merged**